component/wallet-cli/pkg/oidc4vp/oidc4vp_flow.go

```diff
@@ -602,6 +603,21 @@ func (f *Flow) createIDToken(
 	return idTokenJSON, nil
 }
 
+func (f *Flow) extractAdditionalClaims(requestObjectScope string) map[string]Claims {
```

Comment:
As discussed, for now wallet returns random claims and does not considers custom scope.
CC: @sudeshrshetty  @vkubiv

Reply:
@mishasizov-SK  What do you mean by random? Any?

Reply:
I am not sure what you mean, you can pre-configure wallet CLI with pre-defined "scope:claim", and ask the exact scope while testing.

Reply:
@sudeshrshetty updated